torch/_inductor/overrides.py

```diff
@@ -718,6 +718,12 @@ def fuse_unary(gm: torch.fx.GraphModule):
                     computation_node.padding, str
                 ):
                     continue
+                # TODO: support more conv+binary+unary fusion.
+                if type(computation_node) in [
+                    ConvBinary2d,
+                    ConvBinaryInplace2d,
+                ] and type(unary_node) not in [nn.ReLU]:
+                    continue
```

Comment:
What's the behavior of `F.relu`?

Reply:
We don't support it now, only support **nn.module** fusion, for function path, we can enable it at the next step.

Reply:
@XiaobingSuper , I'm okay with this PR to fix TIMM failed model. And let's submit another PR to support `F.ReLU`.